Cargo.toml

```diff
@@ -46,6 +46,7 @@ members = [
 
 [workspace.dependencies]
 aes = "0.8"
+alloy = { version = "0.12.6" , default-features = false, features = ["full"] }
```

Comment:
Why the full features on the whole package level?

Reply:
Yeah. I wanted to add everything for the moment and once the migration is done, reduce it to used features only.

Reply:
OKay, but at the workspace level, you don't have to define any at all.